WineCellar/WineCellar.GUI/RegisterWine.xaml

```diff
@@ -47,13 +53,13 @@
                 </StackPanel>
 
                 <StackPanel Orientation="Horizontal">
-                    <Label Content="Inkoopprijs"/>
+                    <Label Content="Inkoopprijs (€)&#xD;&#xA;"/>
                     <TextBox x:Name="buy" TextWrapping="Wrap" LostFocus="PlaceholderLostFocus" GotFocus="PlaceholderFocus" 
                              Text="Inkoopprijs&#xD;&#xA;" Width="200" Height="25" />
                 </StackPanel>
 
                 <StackPanel Orientation="Horizontal">
-                    <Label Content="Verkoopprijs"/>
+                    <Label Content="Verkoopprijs (€)&#xD;&#xA;"/>
```

Comment:
Heeft deze encoded tekst (dat wat in de property "Content" staat) een functie? Zo niet, graag weghalen om leesbaarheid te verbeteren.